src/app/screens/Receive/index.tsx

```diff
@@ -237,44 +239,72 @@ function Receive() {
       ) : (
-        <form onSubmit={handleSubmit} className="h-full">
-          <fieldset className="h-full" disabled={loading}>
-            <Container justifyBetween maxWidth="sm">
-              <div className="py-4">
+        <div className="pt-4">
+          <form onSubmit={handleSubmit}>
+            <fieldset disabled={loading}>
+              <Container justifyBetween maxWidth="sm">
+                <div className="py-4">
+                  <div className="mb-4">
+                    <DualCurrencyField
+                      id="amount"
+                      min={0}
+                      label={t("amount.label")}
+                      placeholder={t("amount.placeholder")}
+                      fiatValue={fiatAmount}
+                      onChange={handleChange}
+                      autoFocus
+                    />
+                  </div>
+
+                  <div className="mb-4">
+                    <TextField
+                      id="description"
+                      label={t("description.label")}
+                      placeholder={t("description.placeholder")}
+                      onChange={handleChange}
+                    />
+                  </div>
+                </div>
                 <div className="mb-4">
-                  <DualCurrencyField
-                    id="amount"
-                    min={0}
-                    label={t("amount.label")}
-                    placeholder={t("amount.placeholder")}
-                    fiatValue={fiatAmount}
-                    onChange={handleChange}
-                    autoFocus
+                  <Button
+                    type="submit"
+                    label={t("actions.create_invoice")}
+                    fullWidth
+                    primary
+                    loading={loading}
+                    disabled={loading}
                   />
                 </div>
+              </Container>
+            </fieldset>
+          </form>
+          {isAlbyUser ? (
+            <div>
+              <Container justifyBetween maxWidth="sm">
+                <div className="relative flex  items-center mb-4">
+                  <div className="flex-grow border-t border-gray-300 dark:border-gray-700"></div>
+                  <span className="flex-shrink mx-4  text-gray-500 dark:text-gray-400 fw-bold">
+                    {t("or")}
+                  </span>
+                  <div className="flex-grow border-t  border-gray-300 dark:border-gray-700"></div>
+                </div>
 
                 <div className="mb-4">
-                  <TextField
-                    id="description"
-                    label={t("description.label")}
-                    placeholder={t("description.placeholder")}
-                    onChange={handleChange}
+                  <Button
+                    type="button"
+                    label={t("receive_via_bitcoin_address")}
+                    fullWidth
+                    loading={loading}
```

Comment:
when I click create invoice, this button also shows a loading spinner. This is strange IMO

Reply:
resolved